src/ray/object_manager/plasma/dlmalloc.cc

```diff
@@ -1,3 +1,17 @@
+// Copyright 2017 The Ray Authors.
```

Comment:
I think we should exclude plasma code.

Reply:
Yes I think we need to exclude any vendored code like this

Reply:
No problem.